documentation/12-configuration/README.md

```diff
@@ -119,13 +122,15 @@ The format of the JSON file configuration is as follows:
 - `poll` is the time in seconds between polling for health checks.
 - `ttl` is the time-to-live of a successful health check. This should be longer than the polling rate so that the polling process and the TTL aren't racing; otherwise Consul will mark the service as unhealthy.
 - `tags` is an optional array of tags. If the discovery service supports it (Consul does), the service will register itself with these tags.
+- `timeout` an optional value to wait before forcibly killing the health check. Health checks killed in this way are terminated immediately (`SIGKILL`) without an opportunity to clean up their state. This means that a heartbeat will not be sent. The minimum timeout is `1ms`. Omitting this field means that ContainerPilot will wait indefinitely for the health check.
```

Comment:
> Omitting this field means that ContainerPilot will wait indefinitely for the health check.

You probably debated this privately before deciding on it, but would the service's `poll` or `ttl` value be a more sane default? Yes, this default behavior matches the previous behavior, but is that most desirable?